### PR TITLE
Fibonacci

### PR DESCRIPTION
**Pull request by Ptysiek**

### CODE REVIEW

**Comment by Ptysiek** [comment]
Hi! :)
Just making sure if I should change anything?
@ziobron

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,18 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    int a = 0;
+    int b = 1;
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:5]: What is 'a? Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @Ptysiek

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Ptysiek for on-time delivery

**Pull request closed without merging**